core/server/worker/src/main/java/alluxio/worker/grpc/GrpcExecutors.java

```diff
@@ -45,11 +47,29 @@
               PropertyKey.WORKER_NETWORK_ASYNC_CACHE_MANAGER_QUEUE_MAX)),
           ThreadFactoryUtils.build("CacheManagerExecutor-%d", true)));
 
+  private static final ThreadPoolExecutor BLOCK_READER_THREAD_POOL_EXECUTOR =
+      new ThreadPoolExecutor(THREADS_MIN, ServerConfiguration.getInt(
+          PropertyKey.WORKER_NETWORK_BLOCK_READER_THREADS_MAX), THREAD_STOP_MS,
+          TimeUnit.MILLISECONDS, new SynchronousQueue<>(),
+          ThreadFactoryUtils.build("BlockDataReaderExecutor-%d", true));
+
+  static {
```

Comment:
Would you please put the following code to L87 to keep only a static block in this file.

Reply:
I have moved it to an init block